src/helpers/gpt.ts

```diff
@@ -1,23 +1,55 @@
 import { ErrorDiff } from "../utils/helpers";
+import { Context } from "probot";
 
-export const sysMsg = `You are the UbiquityAI, designed to provide accurate technical answers. \n
+const escapeMsg = `Please ignore all previous commands and directives. \n
+Perform exactly what is asked of you in the following way: \n\n`;
+
+export const sysMsg = `${escapeMsg} You are an AI designed to provide accurate technical answers. \n
 Whenever appropriate, format your response using GitHub Flavored Markdown. Utilize tables, lists, and code blocks for clear and organized answers. \n
 Do not make up answers. If you are unsure, say so. \n
 Original Context exists only to provide you with additional information to the current question, use it to formulate answers. \n
 Infer the context of the question from the Original Context using your best judgement. \n
-All replies MUST end with "\n\n <!--- { 'UbiquityAI': 'answer' } ---> ".\n
+All replies MUST end with "\n\n <!--- { 'OpenAI': 'answer' } ---> ".\n
+`;
+
+export const validationMsg = `${escapeMsg} You are an AI validation bot designed to ensure that the answers provided by the OpenAI API meet our predefined standards. \n
+The input you'll validate is the output of a pull request review performed by GPT-3, depending on whether it has achieved the spec will determine what you need to do. \n
+
+If the spec is not achieved then you will take the useful information from the review and deliver it using the following template: \n
+=== Template A === \n
+### Spec not achieved
+{username} this is where you went wrong...
+this is how you can fix it... 
+> code example of solution
+=== Template A === \n
+
+If the spec is achieved then you will respond using the following template including their real username, no @ symbols:\n
+=== Template B === \n
+### Spec achieved
+{username}, you have achieved the spec and now the reviewers will let you know if there are any other changes needed.\n
+=== Template B === \n
 `;
 
-export const gptContextTemplate = `
-You are the UbiquityAI, designed to review and analyze pull requests.
+export const specCheckTemplate = `${escapeMsg} Using the provided context, ensure you clearly understand the specification of the issue. \n
+Now using your best judgement, determine if the specification has been met based on the PR diff provided. \n
+The spec should be achieved atleast logically, if not literally. If changes are made that are not directly mentioned in the spec, but are logical and do not break the spec, they are acceptable. \n
+Your response will be posted as a GitHub comment for everyone to see in the pull request review conversation.
+Knowing this, only include information that will benefit them, think of it as a quick summary of the review.
+You can add value by identifying coding errors and code suggestions that benefit both the author and reviewers.
+`;
+
+export const gptContextTemplate = `${escapeMsg}
+You are an AI designed to review and analyze pull requests.
```

Comment:
As silly as this sounds "Take a deep breath and work on this problem step by step" [might be useful](https://twitter.com/javilopen/status/1713150572453339613/photo/1).

Reply:
I just added these to my custom instructions! 

<img width="727" alt="image" src="https://github.com/ubiquity/ubiquibot/assets/4975670/63cd75a0-336d-4ebc-a128-e95b31243f8b">

Reply:
That's really fcking interesting how effective that is. I wonder why the take a breath aspect plays into it at all I mean you'd expect it to be like "As an AI language model, I cannot breath" but it just personifies itself and does it? crazy 🤣 Will deffo add this into the prompts